Makefile

```diff
@@ -15,13 +15,13 @@ build-bundle:
 build-swagger:
 	cd swagger && npm run build
 test:
-	JEKYLL_NO_GITHUB=true bundle exec htmlproofer ./_site --assume-extension --check-html --allow-hash-href --empty-alt-ignore --only-4xx --disable-external --url_ignore "/esQuery\.config\.js/"
+	bundle exec htmlproofer ./_site --assume-extension --check-html --allow-hash-href --empty-alt-ignore --only-4xx --disable-external --url_ignore "/esQuery\.config\.js/"
 local:
-	JEKYLL_NO_GITHUB=true bundle exec jekyll serve --incremental
+	bundle exec jekyll serve --incremental
 jekyll-build:
-	JEKYLL_ENV=production bundle exec jekyll build
-deploy:
-	rsync -avP --rsync-path="sudo -u www-data rsync" _site/ developers.italia.it:/data/www/developers.italia.it/web/
+	bundle exec jekyll build
```

Comment:
`jekyll-build` task is used only for production purpose, instead we use `local` for development. I think that it is correct to set production environment in jekyll there.
see [here](https://jekyllrb.com/docs/configuration/environments/)